modules/config.xqm

```diff
@@ -41,15 +41,7 @@ declare variable $config:properties := doc(concat($config:app-root, "/properties
 
 declare variable $config:version := config:get-property('version');
 
-declare variable $config:footer := 
-    <div class="footer" xmlns="http://www.w3.org/1999/xhtml">
-      <div style="float:right" class="version">v. {$config:version} for MEI 4.0.0</div>
-      <img 
-           style="border: 0px; vertical-align:middle;" 
-           alt="DCM Logo" 
-           src="/resources/images/dcm_logo_small_white.png"/>
-           2010-2019 Danish Centre for Music Editing | Royal Danish Library, Copenhagen | <a name="www.kb.dk" id="www.kb.dk" href="http://www.kb.dk">www.kb.dk</a> – 2020 ff. MerMEId Community
-    </div>;
+declare variable $config:footer := config:get-property('footer');
```

Comment:
Sorry, but that was too simple ;) 
Now we got an endless recursion because `config:get-property()` has a special case for the key "footer" to look at `$config:footer`